src/templates/shared/requirements_doc.txt

```diff
@@ -0,0 +1,3 @@
+sphinx
```

Comment:
since for the tests we had named the requirements file "test-requirements.txt", wouldn't be worth naming this file "doc-requirements.txt" for consistency?

Reply:
I have done it the other way round and renamed the `test-requirements.txt` to `requirements_test.txt` as this is the more commonly seen format.

Reply:
(in #51 )

Reply:
(this is the convention used in PyAnsys)

Reply:
"requirements_test.txt as this is the more commonly seen format". This is not true actually. There is no common convention and there is nothing that proves that one pattern is more used than the other. The pattern I selected is this <context>-requirements.txt. If you want to change it. You have to do it everywhere including the DockerFiles in all the templates